.github/workflows/build.yml

```diff
@@ -61,17 +76,138 @@ jobs:
               exit 1
               ;;
           esac
-          mkdir artifacts
-          wget -P "artifacts" "$downloadURL"
+          wget -P "temp" "$downloadURL"
+        working-directory: ravendb/scripts/linux/pkg/deb
+
+      - name: Build deb package
+        run: |
+          chmod a+x *.sh
+          case "${{ matrix.arch }}" in
+            "arm32v7")
+              arch="armhf"
+              ;;
+            "arm64v8")
+              arch="arm64"
+              ;;
+            "x64")
+              arch="amd64"
+              ;;
+            *)
+              echo "Unsupported architecture"
+              exit 1
+              ;;
+          esac
+          source ./set-ubuntu-${{ matrix.ubuntuName }}.sh
+          source ./set-raven-platform-${arch}.sh
+          ./build-deb.sh
+          echo "DEB_ARCH=$arch" >> $GITHUB_ENV
+          echo "UBUNTU_VERSION=$DISTRO_VERSION" >> $GITHUB_ENV
+        working-directory: ravendb/scripts/linux/pkg/deb
+        env:
+          RAVENDB_VERSION: ${{ inputs.version }}
+
+      - name: Upload deb package to artficats
+        uses: actions/upload-artifact@v3
+        with:
+          name: package-${{ matrix.ubuntuName }}-${{ matrix.arch }}
+          path: ravendb/scripts/linux/pkg/deb/dist/${{ env.UBUNTU_VERSION }}/ravendb*.deb
+
+      - name: Upload deb package to s3
+        run: |
+          set -e
+          set -x
+          #export DRY_RUN=1

```

Comment:
can you allow passing dry_run as the top-level GA action parameter?

Reply:
Added.